recipes/wilzegers_autotest/all/conanfile.py

```diff
@@ -0,0 +1,40 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class WilzegersAutotestConan(ConanFile):
+    name = "wilzegers_autotest"
+    version = "0.1"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://gitlab.com/wilzegers/autotest"
+    description = "Autotest facilitates the testing of class interfaces"
+    topics = ("autotest", "testing")
+    settings = "compiler"
+
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def validate(self):
+        if self.settings.compiler != 'clang':
+            raise ConanInvalidConfiguration("Only clang allowed")
```

Comment:
Won't it work with gcc@Linux or apple-clang@Macos?

Reply:
Not sure about new version of ```apple-clang```. Library use ```libfuzzer``` which available only in ```clang```

Reply:
How about doing `self.settings.os != "Linux"` then?

Reply:
or `if self.settings.compiler not in ("gcc", "clang")`.

Reply:
Never mind. libfuzzer is clang only atm.